src/main/java/racing/PrintResult.java

```diff
@@ -0,0 +1,36 @@
+package racing;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class PrintResult {
+    public void checkSame(List<String> winners, Car toCompare, Car winner) {
+        if (winner.getMovingRange() == toCompare.getMovingRange() && toCompare != winner) {
+            winners.add(toCompare.getName());
+            System.out.print(", " + toCompare.getName());
+        }
+    }
```

Comment:
```suggestion
    private boolean checkSame(Car toCompare, int winnerPosition) {
        if (winnerPosition == toCompare.getMovingRange()) {
            return true;
        }
        return false;
    }
```
`3개 이상의 인스턴스 변수를 가진 클래스를 쓰지 않는다.`라는 객체지향 생활 체조 원칙을 반영해보면 어떨까요?

`toCompare != winner`로직과 `winners.add(toCompare.getName());`를 제거 할 수 있을 것 같습니다.

Reply:
done